crispy_forms/tests/test_layout.py

```diff
@@ -489,21 +490,12 @@ def test_layout_composition():
     c = Context({"form": SampleForm(), "form_helper": form_helper})
     html = template.render(c)
 
-    assert "multiField" in html
-    assert "formColumn" in html
-    assert 'id="multifield_info"' in html
-    assert 'id="column_name"' in html
-    assert 'class="formColumn columns"' in html
-    assert 'class="buttonHolder">' in html
-    assert 'input type="submit"' in html
-    assert 'name="Save"' in html
-    assert 'id="custom-div"' in html
-    assert 'class="customdivs"' in html
-    assert "last_name" not in html
-
-
-@only_uni_form
-def test_second_layout_multifield_column_buttonholder_submit_div():
+    # Bootstrap 4 does not contain a multifield template
```

Comment:
I think there is value in testing of this layout, so switched to bootstrap3. I'd run for bootstrap3 and 4 but this layout is currently invalid in bootstrap 4. But that's a separate issue/pr.